changes/unreleased/Added-20230814-102123.yaml

```diff
@@ -0,0 +1,3 @@
+kind: Added
```

Comment:
Out of curiosity, what are these files / how are they used? 🤔 
And have they been automatically generated with some command from the Makefile?

Reply:
The workflow is more or less like this:

- all PRs must include an entry in `changes/`
  * using `make install_tools` / `changie new` allows you to add one easily
  * this is checked using CI
- the release process batches and merges these into the CHANGELOG and uses these for the release notes

Overall I'm not too happy with this, so suggestions are welcome...  Maybe going back to manual `changes/` entries would be even better than the current situation. I think one important requirement is being able to discern between the CHANGELOG (user-facing) and the commit log (developer-facing).

Reply:
Thanks for explaining!

I don't have much experience with release-tooling, pretty much only `commitlint` and companions, which aren't great, but at least it's clear and there's no "manual" process to it apart from writing commit messages 😄 

That said, I don't have any experience with the current workflow, so I don't really know what to suggest :)